nuxeo-features/nuxeo-platform-oauth/src/main/java/org/nuxeo/ecm/platform/oauth2/clients/OAuth2ClientService.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Thomas Roger
+ *
+ */
+package org.nuxeo.ecm.platform.oauth2.clients;
+
+/**
+ * @author <a href="mailto:ak@nuxeo.com">Arnaud Kervern</a>
```

Comment:
I am wondering, do we really care about contributors (on the copyright)? Wouldn't be "Nuxeo" enough as contributors?
I'm bored to think to update/move the contributors each time :)
WDYT @efge?